app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -8,10 +8,10 @@
 }
 
 .gem-c-feedback--top-margin {
-  margin-top: $gutter;
+  margin-top: govuk-spacing(6);
 
-  @include media(tablet) {
-    margin-top: $gutter * 2;
+  @include govuk-media-query($from: tablet) {
+    margin-top: govuk-spacing(6) * 2;
```

Comment:
```suggestion
    margin-top: govuk-spacing(9);
```
perhaps?

Reply:
Yup, I always forget what's the highest number on the scale. Will update